calender/calendar_tui.py

```
@@ -383,7 +382,7 @@ def __init__(self, stdscr, mcp_client: MCPClient, timezone: str = "America/New_Y
                 # Define color 8 as dark red (RGB values scaled to 0-1000)
                 curses.init_color(8, 545, 0, 0)  # Dark red
                 conflict_color = 8
-            except:
+            except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)